datafusion/core/tests/user_defined/user_defined_scalar_functions.rs

```diff
@@ -341,6 +341,43 @@ async fn case_sensitive_identifiers_user_defined_functions() -> Result<()> {
     Ok(())
 }
 
+#[tokio::test]
+async fn test_user_defined_functions_with_alias() -> Result<()> {
```

Comment:
Maybe useful to test again the case where you create a udf1 with an alias and then a different udf with the same alias? I don't know what's the expected behavior there

Reply:
the behavior will like `HashMap::insert`, the old value of same alias will be updated with new udf